karma.conf.js

```diff
@@ -33,8 +33,12 @@ module.exports = (config) => {
               path.join(__dirname, 'test'),
               require.resolve('airbnb-js-shims'),
             ],
-            query: {
-              presets: ['airbnb'],
+            options: {
+              presets: [
+                'airbnb',
+                // transform to cjs so sinon can stub properly
```

Comment:
interesting, does the airbnb preset not already do this? i think it accepts a modules option too

Reply:
I originally tried to set it on the airbnb preset, but it prevents you from specifying anything other than true/false/auto (https://github.com/airbnb/babel-preset-airbnb/blob/master/index.js#L35)

Reply:
K, I’ll have to expand that to accept cjs; in the meantime, let’s set this one to false so it doesn’t include the env preset twice?

Reply:
Sure thing 👍